example/satosa/openid4vci_frontend.yaml

```diff
@@ -0,0 +1,731 @@
+      class: ParHandler
+      path: '/par'
+    credential_offer:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.credential_offer_endpoint
+      class: CredentialOfferHandler
+      path: '/credential'
+    credential_offer_qrcode:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.credential_offer_qrcode_endpoint
+      class: CredentialOfferQrCodeHandler
+      path: '/credential-qrcode'
+    authorization:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.authorization_endpoint
+      class: AuthorizationHandler
+      path: '/authorization'
+    token:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.token_endpoint
+      class: TokenHandler
+      path: '/token'
+    nonce:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.nonce_endpoint
+      class: NonceHandler
+      path: '/nonce-endpoint'
+    credential:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.credential_endpoint
+      class: CredentialHandler
+      path: '/credential'
+#    deferred_credential:
+#      module: pyeudiw.satosa.frontends.openid4vci.endpoints.deferred_credential_endpoint
+#      class: DeferredCredentialHandler
+#      path: '/deferred-credential'
+#    notification:
+#      module: pyeudiw.satosa.frontends.openid4vci.endpoints.notification_endpoint
+#      class: NotificationHandler
+#      path: '/notification'
+    metadata:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.metadata_endpoint
+      class: MetadataHandler
+      path: '/.well-known/openid-federation'
+    status_list:
+      module: pyeudiw.satosa.frontends.openid4vci.endpoints.status_list_endpoint
+      class: StatusListHandler
+      path: '/status' &status_list_endpoint_path
+
+  qrcode:
+    ui_template: "qr_code.html"
+    size: 250 # px
+    color: '#000000'  # hex
+    expiration_time: 120 # seconds
+    logo_path: 'wallet-it/wallet-icon-blue.svg' # relative to static_storage_url
+
+  network:
+    httpc_params: &httpc_params
+      connection:
+        ssl: true
+      session:
+        timeout: 6
+
+  jwt:
+    default_sig_alg: ES256  # or RS256. Please note that this signature alg MUST be compliant with the private keys used for the signature. X.509 certificates MUST be therefore ECDSA using ES, and RSA using RS
+    default_enc_alg: RSA-OAEP
+    default_enc_enc: A256CBC-HS512
+    default_exp: 6         # minutes
+    enc_alg_supported:
+      - RSA-OAEP
+      - RSA-OAEP-256
+      - ECDH-ES
+      - ECDH-ES+A128KW
+      - ECDH-ES+A192KW
+      - ECDH-ES+A256KW
+    enc_enc_supported:
+      - A128CBC-HS256
+      - A192CBC-HS384
+      - A256CBC-HS512
+      - A128GCM
+      - A192GCM
+      - A256GCM
+    sig_alg_supported:
+      - RS256
+      - RS384
+      - RS512
+      - ES256
+      - ES384
+      - ES512
+    access_token_exp: 90 # minutes
+    refresh_token_exp: 120 # minutes
+    par_exp: 90 # minutes
+
+  # private jwk
```

Comment:
please add an HUGE comment with a WARNING about the importance to change these keys and the instruction to do this, for security reason

please do the same in the openid4vp example configuration